src/DIRAC/WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1250,6 +1250,12 @@ def __checkAndPrepareJob(self, jobID, classAdJob, classAdReq, owner, ownerDN,
     if vo:
       classAdReq.insertAttributeString('VirtualOrganization', vo)
 
+    inputDataPolicy = Operations().getValue('InputDataPolicy/InputDataModule')
+    if inputDataPolicy and not classAdJob.lookupAttribute('InputDataModule'):
+      classAdJob.insertAttributeString('InputDataModule', inputDataPolicy)
+
+    # ################## adding DIRAC/VOPolicy as classAds
+    # FIXME: to remove
```

Comment:
What's the plan for removing this? The next patch release or v7r3?

Reply:
v7r3.